experimental/dds/tree2/src/feature-libraries/chunked-forest/chunkedForest.ts

```diff
@@ -110,17 +113,9 @@ class ChunkedForest extends SimpleDependee implements IEditableForest {
 				);
 				return this.mutableChunkStack[this.mutableChunkStack.length - 1];
 			},
-			moveIn(
-				index: number,
-				toAttach: DetachedField,
-				invalidateDependents: boolean = true,
-			): number {
-				if (invalidateDependents) {
-					this.forest.invalidateDependents();
```

Comment:
I think the invalidation is missing here.

Reply:
this logic used to handel when it was called from the visitor vs called directly. if this is only called from the code below now, having it as a method on the vistor with no documentation and a similar style name really makes it look like one of the interface methods. A doc comment would help (also one on the other private helpers)

Reply:
Resolving cause this method was removed with the change to the DeltaVisitor API.